WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.Shared/HoloLens/HolographicPerception.cs

```diff
@@ -57,10 +57,70 @@ public enum SimulationControlMode
         }
 
         /// <summary>
+        /// Enumeration defining the priority levels for the Holographic Perception Simulation control stream.
+        /// </summary>
+        public enum SimulationControlStreamPriority
+        {
+            /// <summary>
+            /// Low priority.
+            /// </summary>
+            Low = 0,
+
+            /// <summary>
+            /// Normal priority.
+            /// </summary>
+            Normal
+        }
+
+        /// <summary>
+        /// Creates a simulation control stream.
+        /// </summary>
+        /// <param name="priority">The control stream priority.</param>
+        /// <returns>PerceptionSimulationControlStreamId containing the identifier of the created stream.</returns>
+        /// <remarks>This method is only supported on HoloLens.</remarks>
+        public async Task<PerceptionSimulationControlStreamId> CreatePerceptionSimulationControlStream(SimulationControlStreamPriority priority)
+        {
+            if (!Utilities.IsHoloLens(this.Platform, this.DeviceFamily))
+            {
+                throw new NotSupportedException("This method is only supported on HoloLens.");
+            }
+
+            string payload = string.Format(
+                "priority={0}",
+                (int)priority);
+
+            return await this.GetAsync<PerceptionSimulationControlStreamId>(
+                HolographicSimulationStreamApi,
+                payload);
+        }
+
+        /// <summary>
+        /// Deletes a simulation control stream.
+        /// </summary>
+        /// <param name="streamId">The identifier of the stream to be deleted.</param>
+        /// <returns>Task tracking completion of the REST call.</returns>
+        /// <remarks>This method is only supported on HoloLens.</remarks>
+        public async Task DeletePerceptionSimulationControlStream(string streamId)
```

Comment:
Can this be a PerceptionSimulationControlStreamId param instead of/in addition to a string? I've been trying to keep inputs and outputs as the same objects rather than using the underlying types.